core/src/main/scala/chisel3/CompileOptions.scala

```diff
@@ -6,25 +6,47 @@ import scala.language.experimental.macros
 import scala.reflect.macros.blackbox.Context
 
 trait CompileOptions {
+  // ==== EASY TO MIGRATE ====
+
+  // Should the reset type of Module be a Bool or a Reset
+  // 1) Set migrateInferModuleReset=true, fix errors by mixing in [[RequireSyncReset]])
```

Comment:
```suggestion
  // 1) Set migrateInferModuleReset=true, fix errors by mixing in [[RequireSyncReset]]) or by using a CompileOptions that has inferModuleReset = true
```

In practice I think I will do more of the latter